pkg/process/config/config_test.go

```diff
@@ -670,6 +665,94 @@ func TestProcessDiscoveryInterval(t *testing.T) {
 	}
 }
 
+func TestGetAPIEndpoints(t *testing.T) {
```

Comment:
Is `pkg/process/config` the most natural place for this test, we probably only use these endpoints when we initialize the forwarder in the `main` package.

Reply:
Per our discussion on slack, I moved the test to `cmd/process-agent/config_test.go` and removed `APIEndpoints` from `AgentConfig`